src/ewatercycle/esmvaltool/diagnostic/copier.py

```diff
@@ -1,13 +1,19 @@
 """ESMValTool diagnostic script that copies the preprocessing output to the diagnostic output."""
+
 import logging
 import shutil
 from pathlib import Path
 
-from esmvaltool.diag_scripts.shared import (
-    ProvenanceLogger,
-    get_diagnostic_filename,
-    run_diagnostic,
-)
+try:
```

Comment:
Moving this to where it's used (i.e. `def *Forcing.generate(...)`) would allow users to mostly use ewatercycle without esmvaltool.